src/app/features/tech-record/components/tech-record-title/tech-record-title.component.ts

```diff
@@ -18,7 +18,7 @@ import { Observable, take } from 'rxjs';
 })
 export class TechRecordTitleComponent implements OnInit {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  @Input() vehicle?: V3TechRecordModel | any;
+  @Input() vehicle?: any;
```

Comment:
how come we had to down grade this?

Reply:
This isn't a downgrade, the new eslint package is much more powerful and basically says 'any subsumes the union' or something - basically if you or an any it doesn't matter, the type will always be any.